### ISSUE TITLE
[Suggestion] Pride Banner Patterns

### ISSUE THREAD

**Issue by ChloeDawn** [gamma-delta/HexMod#212]
It would be pretty great if the pride pigments could be used in a loom to apply their respective flag pattern to banners.

**Comment by yrsegal** [comment]
Problem: Banner patterns can be colorized. How would this be handled?

**Comment by ChloeDawn** [comment]
Good point, that was an oversight on my part. An alternative is they could be treated as dyes, creating pride variants of banner bases, and potentially applying patterns that are pride colored too? The latter half of that may be more trouble than it's worth though.
